database/node.go

```diff
@@ -34,6 +34,7 @@ func NewNode(cfg *Config, consensusAPI conAPI.API, registry *prometheus.Registry
 		return nil, err
 	}
 
+	// Todo: Should be refactored, creating a new instance of a struct SHOULDN'T do any FS changes
```

Comment:
This comment could be part of the issue added to the backlog refering to the pending refactor

Reply:
done https://github.com/lightstreams-network/lightchain/pull/88/commits/802bf83493a760dae603bcba3875780755e846ac